src/components/forms/DocumentUploadForm/shared/FileSelected/FileAsImage.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="document-upload-preview">
```

Comment:
Component named as `FileAsImage`, root class is `document-upload-preview`.
I think, if we have common classes – it should be isolated presentation component.
Each component should has its own classes or global helpers/utilities. When we are using classes from other components – this is a big error because our components are starting be binded to each other.

Reply:
my fault, did not rename classes after rename component